README.md

```diff
@@ -1,32 +1,50 @@
+
+_🚧 **Note:** This project is in the early stages of development_
 
-🚧 **Note:** This project is in the early stages of development, but it does contain useful functionality for headless WordPress sites like plugins and npm packages that assist in authentication and previews.
 
 ## Quick Start
 
-Eager to try out the Headless Framework? Here's how you can get started with our Preview example:
+Eager to try out the Headless Framework? Here's how you can get started:
 
 1. Create a WordPress site if you haven't already. We recommend using [Local](https://localwp.com/)!
-2. Download, upload, and activate the `wpe-headless` plugin in this repository. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
+2. Download, upload, and activate the `wpe-headless` plugin. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
 3. Install [WP GraphQL](https://wordpress.org/plugins/wp-graphql/) on the WordPress site if it's not already installed
-4. [Clone this repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) to a directory of your choice
-5. Navigate to `examples/preview` in the cloned repository
+4. Download our [Getting Started example project](https://github.com/wpengine/headless-framework/tree/canary/examples/getting-started) and navigate to it locally
 6. `cp .env.local.sample .env.local`
 7. Populate `NEXT_PUBLIC_WORDPRESS_URL` and `WPE_HEADLESS_SECRET` accordingly in `.env.local`
 8. `npm install && npm run dev`
 
 ## Framework Features
 
-- Headless Auth Flows
-  - OAuth token authentication for users
-  - Auth handler for Express/Next that exchanges a code for an access token. The access token can be used to make authenticated calls to WordPress via WPGraphQL or REST.
-- [Previews](./docs/previews/README.md)
-  - Rewrite preview and draft links in WP Admin to redirect to the frontend.
+### Plugin Features
+
+- **[Headless post previewing](./docs/previews/README.md)**
+  - OAuth token authentication creation
+  - Preview and draft link rewrites in WP Admin to redirect to the front-end
+- **Smart content redirects**
+  - Automatically redirects content from the WP site to the front-end site to minimize site visitors’ confusion and avoid SEO penalties for duplicate content
+  - Redirects hyperlinks inserted into posts’ content to the front-end site
+- **Removal of WP themes**
+  - Prevents confusion around the active theme
+- **Ability to define custom menus in a GUI**
+- **Additional data exposed through WPGraphQL**
+  - Block stylesheets
+
+### npm Package Features
```

Comment:
Need to flesh the npm features out more for sure

Reply:
I added some suggested descriptions for `HeadlessProvider` and `TemplateLoader` below.

Reply:
Thank you!